docs/API.md

```diff
@@ -1,8 +1,57 @@
 # API Documentation
 
+
+## /search
+### Description
+The name for this API endpoint is `/search`. This endpoint uses the other three endpoints below to query their respective collections. 
+
+### Calling the Endpoint
+`/search?term=SEARCH_PARAMETER`
+To call this endpoint, make a request using the example above. When replacing the `term` to be searched, it is important to point out that all search terms are case sensitive. 
```

Comment:
Let's try and add some highlight to the fact that these are CASE SENSITIVE. It's a small thing that can cause big issues if missed.